src/convertToObject.js

```diff
@@ -6,7 +6,35 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // This onbject will be returned.
+  const styledObject = {};
+
+  // Who the hell write sourceString like that? My grandma can do better...
+  // If i ever see this during actuall job, i'll leave the company.
```

Comment:
[CODE STYLE] Comments should be used to explain the purpose of the code and provide information that is not immediately obvious from the code itself. This comment doesn't provide any useful information and can be removed.

Reply:
You just don't understand the joke.